Phonebook.Frontend/src/app/pages/settings/settings.component.ts

```diff
@@ -47,30 +51,67 @@ export class SettingsComponent implements OnInit, OnDestroy {
     this.layoutValue = layout;
   }
 
+  public getThemeName(theme: Theme) {
+    switch (theme) {
+      case Theme.blue_light_theme: {
+        return this.i18n({
+          value: "Blue Light Theme",
+          description: "Color Theme Option: Blue Light",
+          id: "SettingsComponentColorThemeBlueLight",
+          meaning: "SettingsComponent"
+        });
+      }
+      case Theme.blue_dark_theme: {
+        return this.i18n({
+          value: "Blue Dark Theme",
+          description: "Color Theme Option: Blue Dark",
+          id: "SettingsComponentColorThemeBlueDark",
+          meaning: "SettingsComponent"
+        });
+      }
+      case Theme.magenta_light_theme: {
+        return this.i18n({
+          value: "Magenta Light Theme",
+          description: "Color Theme Option: Magenta Light",
+          id: "SettingsComponentColorThemeMagentaLight",
+          meaning: "SettingsComponent"
+        });
+      }
+      case Theme.magenta_dark_theme: {
+        return this.i18n({
+          value: "Magenta Dark Theme",
+          description: "Color Theme Option: Magenta Dark",
+          id: "SettingsComponentColorThemeMagentaDark",
+          meaning: "SettingsComponent"
+        });
+      }
```

Comment:
Hier sollte noch ein  `default` case rein in dem ein Error geworfen wird.

Reply:
`      default: {
        return this.i18n({
          id: 'GeneralErrorMessage',
          value: 'Something went wrong. Please try again.'
        });
      }`

Das würde vermutlich nicht ausreichen, bzw macht das überhaupt Sinn?

Reply:
Nein ich meine ein `throw Error('Translation for theme ${theme} does not exists')`

Reply:
Dieser Error hat auch ein recht auf Übersetzung.

Reply:
Mhm, bin da nicht deiner Meinung, da er in Produktion nie auftreten sollte. Ist nur dafür da falls mal jemand vergisst etwas anzupassen.

Reply:
und dann ist er doch in produktion ^^ eseln ist mänchlich

Reply:
Ja, aber da es kein Fehler ist den ein Nutzer sehen sollte. Finde ich sollte man ihn auch nicht übersetzen. Ist ja nicht so daß ihm der Fehler weiterhelfen würde, wie bei z.B. "Die Person mit der ID ... Existiert nicht." 
Übersetzt man ihn und der Nutzer hat es auf deutsch eingestellt macht es das sogar schwieriger den Fehler zu finden.

Reply:
Es ist halt ein Javascript Error und kein Anwendungsfehler.